media/js/base/mozilla-utils.js

```diff
@@ -33,8 +33,8 @@ if (typeof window.Mozilla === 'undefined') {
         switch(platform) {
         case 'windows':
             data.os = 'Desktop';
-            data.name = site.isARM ? 'Windows ARM64/AArch64' : 'Windows 32-bit';
-            data.version = site.isARM ? 'win64-aarch64': 'win';
```

Comment:
Are we no longer detecting `isARM` for windows? If so, do we need to remove-- https://github.com/mozilla/bedrock/blob/74f53d34a9a2513111cf9fe9432817f079d290a6/tests/unit/spec/base/mozilla-utils.js#L90-L94